lib/client.js

```diff
@@ -0,0 +1,169 @@
+var WS = require('ws');
+var debug = require('debug')('docker-exec-websocket-server:lib:client');
+var url = require('url');
+var events = require('events');
+var _ = require('lodash');
+var assert = require('assert');
+var msgcode = require('../lib/messagecodes.js');
+var through = require('through');
+
+export default class DockerExecWebsocketClient extends events.EventEmitter {
+  constructor (options) {
+    super();
+    this.options = _.defaults({}, options, {
+      tty: true,
+      command: ['/bin/bash'],
```

Comment:
I think it's safer to default to just `sh`, not even `/bin/sh` because it'll be able to find it in the `PATH`.

Anyways, details :) we should also specify a command but a good default is nice :)